authorizations/authorization-chain/src/main/java/org/apache/gravitino/authorization/chain/ChainedAuthorizationPlugin.java

```diff
@@ -63,6 +65,16 @@ private void initPlugins(String catalogProvider, Map<String, String> properties)
               Map<String, String> pluginConfig =
                   chainedAuthzProperties.fetchAuthPluginProperties(pluginName);
 
+              for (Map.Entry<String, String> entry : properties.entrySet()) {
+                if (!entry
+                    .getKey()
+                    .startsWith(ChainedAuthorizationProperties.CHAIN_PLUGINS_PROPERTIES_KEY)) {
+                  pluginConfig.put(entry.getKey(), entry.getValue());
+                }
+              }
+              pluginConfig.put(
+                  BaseAuthorization.UUID, pluginConfig.get(BaseAuthorization.UUID) + pluginName);
+
```

Comment:
If you went pass `hadoop.security.authentication` properties to the plugin, you can use `authorization.chain.hdfs-xxx.hadoop.security.authentication`.
Another thing is, I found you added some new properties key in the ranger-hive-plugin and ranger-hdfs-plugin, please update relation doc. thanks.

Reply:
Done.